modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc

```diff
@@ -66,8 +66,9 @@ spec:
   kataMonitorImage: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:1.2.0
   kataConfigPoolSelector:
     matchLabels:
-      <label_key>: '<label_value>'
+      <label_key>: '<label_value>' <1>
 ----
+<1> Labels in `kataConfigPoolSelector` only support single values; `nodeSelector` syntax is not supported.
```

Comment:
This renders strange. I wonder if it is picking up the ---- from line 70 maybe?

![Screenshot 2023-02-23 at 7 38 29 AM](https://user-images.githubusercontent.com/85293752/220908126-ff34f0d8-b3fd-4198-a04f-8fbde8074e16.png)

Reply:
@opayne1  Figured out the issue. The preview didn't update automatically, and this was a mistake in a previous version that was fixed during peer review.  Manually generating an updated preview now.